encryption/classic-mceliece/classic-mceliece-kem/impl/avx.yaml

```diff
@@ -0,0 +1,4 @@
+type: reference
```

Comment:
No, an implementation that uses avx instructions is what we would categorize as `optimized`